CONTRIBUTING.md

```diff
@@ -174,23 +174,29 @@ When submitting the PR remember to label it with the 📖 (:book:) icon.
 
 ## Releases
 
-Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track releases. Issues in a release milestone have been prioritized and accepted for the release. However, these issues are not committed to the release, unless they are marked as `kind/release-blocking`. Getting them into the release is dependent on someone in the community getting assigned to the issue and completing the work.
-
 - Minor versions CAN be planned and scheduled for each quarter, or sooner if necessary.
   - Each minor version is preceded with one or more planning session.
   - Planning consists of one or more backlog grooming meetings, roadmap amendments,
     and CAEP proposal reviews.
-- Patch versions CAN be planned and scheduled each month for each of the currently supported series (usually N and N-1).
-- Code freeze is in effect 72 hours (3 days) before a release.
+  - Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track work
+    for minor releases. 
+  - Adding an issue to a milestone provides forward visibility on what the next release will be, so, as soon as there
+    is the intent to work on an issue for a target release, contributors are expected to work with maintainers to 
+    set the milestone on the issue for all the new features, API changes, behaviour changes or bug fixes that
+    could be relevant for the Cluster API's users. 
+  - Before adding an issue to a release milestone, maintainers must ensure that the issue have been triaged and
+    there is an assignee who expressed the intent to complete the work before the release date; getting those issue
+    implemented into the release is dependent on completing such work.
+- Code freeze is in effect 72 hours (3 days) before a major/minor release.
   - Maintainers should communicate the code freeze date at a community meeting preceding the code freeze date.
   - Only critical bug fixes may be merged in between freeze & release.
     - Each bug MUST be associated with an open issue and properly triaged.
     - PRs MUST be approved by at least 2 project maintainers.
       - First approver should `/approve` and `/hold`.
       - Second approver should `/approve` and `/hold cancel`.
   - [E2E Test grid](https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api#capi%20e2e%20tests) SHOULD be green before cutting a release.
+- Patch versions CAN be planned and scheduled each month for each of the currently supported series (usually N and N-1).
```

Comment:
nit: I think based on our current policy it comes down to only N as long as we don't bump the contract (which we didn't do for a while).

Example:
* Right now:
    * v1.0 => out of support
    * v1.1 => patch releases
    * v1.2 is not out yet => no release obviously
* After v1.2 release
    * v1.1. support is immediately dropped based on policy
    * v1.2 => patch releases

Reply:
fixed